moveit_kinematics/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(moveit_kinematics)
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED cmake_modules)
```

Comment:
Why is this needed? I seem to recall that using `cmake_modules` for Eigen is deprecated these days. Besides that I don't see other changes to match with this. If this was needed in order to compile on your system there may be more wrong.